.github/workflows/e2e.yaml

```diff
@@ -52,10 +52,10 @@ jobs:
         run: |
           # Change node name in config file
           podman exec -it hirte-agent-bar \
-          sed -i "s/\(Name=\)\(.*\)/\1bar/" /hirte/example-hirte-agent.ini
+          sed -i "s/\(NODE_NAME=\)\(.*\)/\1bar/" /hirte/example-hirte-agent.ini
```

Comment:
I would advise for better test e2e coverage should be done through config files and not the ini.

ini files could be tested as part of code unitests
At the end hirte should run as a service on each node, isnt it?

Reply:
This will be solved in some of the follow up PRs which will rewrite e2e to use RPM and then update regular config files such as `/etc/hirte-agent.conf`